### PR TITLE
Fixes #6097: Add support to private redmine in rudder-dev

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6097

### CODE REVIEW

**Comment by Kegeruneku** [comment]
Looks good to me :)

File: scripts/rudder-dev/rudder-dev [hunk 1]
```diff
@@ -16,6 +16,13 @@ Usage:
     rudder-dev merge all
     rudder-dev merge <first_branch> <next_branch>
     rudder-dev merge <first_branch>
+    rudder-dev <smart_arg> [<PR_message>]
+
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: That was merged in #32, why does it figure here too

Comment (peckpeck) [inline scripts/rudder-dev/rudder-dev]: Because I based it on the then current master which didnt include

**Comment by peckpeck** [comment]
@VinceMacBuche rebased

File: scripts/rudder-dev/rudder-dev (hunk 1 above)
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: Current master incluse that fix, here is my misunderstanding

**Pull request merged**